lib/common/JSONImporter/NodeChangeSet.ts

```diff
@@ -25,4 +27,17 @@ export class NodeChangeSet implements ChangeSet {
             diffObj.value
         );
     }
+
+    validated(fn: (ch: NodeChangeSet) => boolean, errMsg: string): Result<NodeChangeSet, Error> {
+        if (fn(this)) {
+            return Result.Ok(this);
+        } else {
+            const err = new Error(errMsg);
+            return Result.Error(err);
+        }
+    }
+
+    asResult(): Result<NodeChangeSet, Error> {
```

Comment:
Using the above approach, this shouldn't be needed either

Reply:
Removed this function.